core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -156,7 +156,9 @@ public Boolean apply(Void input) {
         try {
           return fs.getStatus(uri).isPersisted();
         } catch (Exception e) {
-          throw Throwables.propagate(e);
+          Throwables.propagateIfPossible(e);
```

Comment:
`Throwables.throwIfUnchecked` is unavailable in Guava 14.0.1 which Alluxio is using now. Maybe we should simply use `throw new RuntimeException(e)` here.